.github/workflows/lint.yml

```diff
@@ -0,0 +1,23 @@
+name: Lint Notes
+
+on:
+  push:
+    branches: "**"
+  pull_request:
+    branches: "**"
+  workflow_dispatch:
+jobs:
+  run-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: MarkdownLint
+        uses: github/super-linter@v4
+        env:
+          VALIDATE_ALL_CODEBASE: true
+          DEFAULT_BRANCH: main
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why is this needed?

Reply:
Oops! Didn’t see that in the default config.